### PR TITLE
STOR-200 Add prepare and verify pods for mysql

### PR DESCRIPTION
**Pull request by stgleb**

### CODE REVIEW

File: io-apps/sysbench/sysbench-prepare.dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM severalnines/sysbench
```
Comment (disrani-px) [inline io-apps/sysbench/sysbench-prepare.dockerfile:1]: Can we have just one container with both the scripts and call either verify.sh or prepare.sh from the pod?

Comment (disrani-px) [reply io-apps/sysbench/sysbench-prepare.dockerfile:1]: We don't even need a separate container IMO. We should just use the sysbench container and put the command in the pod spec. That way we don't need to maintain this extra container

Comment (stgleb) [reply io-apps/sysbench/sysbench-prepare.dockerfile]: Done, I've used only vanila sysbench and mysql client images.

**Comment by stgleb** [comment]
Job https://jenkins.portworx.co/job/Stork/job/stork-2-dev-test/84

**Pull request merged**